readme.md

```diff
@@ -1,19 +1,37 @@
 # GitHub Actions Kubernetes
 
-A collection of GitHub actions for Kubernetes projects.
+A collection of GitHub actions for Kubernetes.
 
-## Example Playbook
+## Setup DigitalOcean
+
+Setup Digital-Ocean CLI and configure Kubernetes.
+
+### Example
```

Comment:
```suggestion
## Actions

### Setup DigitalOcean

Setup DigitalOcean CLI and configure kubectl.

Example:
```

What do you think about this structure? I saw when I added some actions by myself, that the descriptions of the actions where on the same level as the versioning. I found it a bit confusing.